Sources/Layouts/ButtonLayout.swift

```diff
@@ -46,6 +46,25 @@ open class ButtonLayout<Button: UIButton>: BaseLayout<Button>, ConfigurableLayou
         super.init(alignment: alignment, flexibility: flexibility, viewReuseId: viewReuseId, config: config)
     }
 
+    init(type: ButtonLayoutType,
+         title: String,
+         image: ButtonLayoutImage = .defaultImage,
+         font: UIFont? = nil,
+         contentEdgeInsets: UIEdgeInsets? = nil,
+         alignment: Alignment = ButtonLayoutDefaults.defaultAlignment,
+         flexibility: Flexibility = ButtonLayoutDefaults.defaultFlexibility,
+         viewReuseId: String? = nil,
+         viewClass: Button.Type? = nil,
```

Comment:
Are you asking why make new internal initializers instead of adding viewClass parameter to the public initializers?

It's a very minor point, but there already are many optional parameters on the public intializer so I have a slight preference for not adding another one that most people won't need.